aiida/backends/tests/cmdline/commands/test_code.py

```diff
@@ -192,14 +192,23 @@ def test_relabel_code_full_bad(self):
         result = self.cli_runner.invoke(relabel, [str(self.code.pk), 'new_code@otherstuff'])
         self.assertIsNotNone(result.exception)
 
-    def test_delete_one(self):
+    def test_code_delete_one(self):
         result = self.cli_runner.invoke(delete, [str(self.code.pk)])
         self.assertIsNone(result.exception, result.output)
 
         with self.assertRaises(NotExistent):
             from aiida.orm import Code
             Code.get_from_string('code')
 
+    def test_code_delete_one_force(self):
```

Comment:
If we make the check in the command itself (see my comment below) then we can also add a test that when deleting a code with at least one calculation without the force flag will exit with a critical message